sdks/python/run_pylint.sh

```diff
@@ -29,6 +29,12 @@ BASE_BRANCH=python-sdk
 set -e
 set -o pipefail
 
+# This script runs pylint on changed files only and uses git for that purpose.
+# Skip this step if git is not available or this is not a valid git repository.
+(command -v git && git remote) || {
+  echo "Not running pylint. Not in a git repository."; exit 0; 
```

Comment:
I reverted my change about removing the hardcoded remote. It did not work with the testing setup. Testing setup have the remote `origin` and my attempt to get the remote for the current branch did not work.

Another option could be that we can run 'git remote' and default to the first item on the list. Let me know what you think.